python/ray/serve/backend_state.py

```diff
@@ -746,15 +834,12 @@ def _scale_backend_replicas(
             backend_info.backend_config.
             experimental_graceful_shutdown_timeout_s)
 
-        stopped = self._stop_wrong_version_replicas(
-            self._replicas[backend_tag], target_replicas, target_version,
-            graceful_shutdown_timeout_s)
-        if stopped > 0:
```

Comment:
intentional?

Reply:
yes this is now logged inside of the method